### PR TITLE
Build docs using github actions

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Build docs using github actions

#### Why?

To get rid of travis

### CODE REVIEW

**Comment by luca-rath** [comment]
The job somehow doesn't run here, but https://github.com/luca-rath/sulu-docs/pull/1/checks?check_run_id=1473055233 here is the result of the job in my fork

File: .github/workflows/build-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+    docs:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: ammaraskar/sphinx-action@master
+              with:
+                  docs-folder: "./"
+                  build-command: "sphinx-build -nW --keep-going -b html -d _build/doctrees . _build/html"
```
Comment (nnatter) [inline .github/workflows/build-docs.yaml:13]: would you mind to separate the `-nW` into `-n -W`? i just searched for a `-nW` option for about 10 minutes before i understood that these are two options 🙈 

```suggestion
                  build-command: "sphinx-build -n -W --keep-going -b html -d _build/doctrees . _build/html"
```

Comment (luca-rath) [reply .github/workflows/build-docs.yaml]: of course 😄

**Review by nnatter — APPROVED** [review]
Looks good! The `actions/upload-artifact` is nice too 🙂

**Pull request merged**